install_config/configuring_authentication.adoc

```diff
@@ -214,6 +214,14 @@ user API obejct.
 If you are using the `lookup` mapping method, use the following steps for each user after configuring
 the identity provider:
 
+[NOTE]
+====
+{product-title} user names containing `/` are not supported. Identity provider
+user names containing `/` may be supported in a future release by automatically
+mapping the `/` character to a different character when determining the
+{product-title} user name.
+====
+
```

Comment:
This should be the overall rule for all identity providers, not onl for Manually Provisioning a User When Using the Lookup Mapping Method.